HTML/task5/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Lato", sans-serif;
+}
+h1 {
+    font-size: 45px;
+    margin: 30px auto;
+    text-align: center;
+}
+h2 {
+    margin: 5px 0;
+    font-family: inherit;
+}
+input {
+    display: none;
+}
+.label__font {
+    margin: 0 20px 0 0;
+    padding: 4px;
+    border: 1px solid black;
+    border-radius: 4px;
+}
+.label__font:hover {
+    cursor: pointer;
+}
+input[id="arial"]:checked ~ .container {
+    font-family: Arial;
+}
+input[id="arial"]:checked + label {
+    background-color: rgb(165, 160, 160);
+    border: 1px solid transparent;
+}
+input[id="18px"]:checked ~ .container article {
```

Comment:
maybe it's better to set some [data attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) and reach the value by [getter](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes#CSS_access) instead of using `id` to pass data

Reply:
why not use `em` here, you can change for root element and all children will change font-size